src/styles.css

```diff
@@ -1,5 +1,6 @@
 /* You can add global styles to this file, and also import other style files */
 
+
 h1, h2, h3, h4, h5 {
   color: #f68e1e !important;
```

Comment:
Using `important` is dangerous, instead you can use `CSS specificity` which is more powerful.